src/appendices/hed.md

```diff
@@ -168,7 +239,8 @@ Data-feature, sc:Eye-blink-artifact, sc:Seizure-PNES
 ##### Single unpartnered library schema example
 
 If only one schema is being used for annotation, the prefix can be omitted entirely.
-The following `dataset_description.json` indicates that only the HED-SCDORE library schema version 1.0.0 will be used for HED annotation in this dataset.
+The following `dataset_description.json` indicates that only the HED-SCDORE library schema version
+1.0.0 will be used for HED annotation in this dataset.
```

Comment:
The line below this JSON has **annotations** misspelled.

Reply:
Corrected, see this [commit](https://github.com/bids-standard/bids-specification/pull/1592/commits/1f08baf09c3031f3563f08c136a554a67f6dbe10).